packer/rules.bzl

```
@@ -66,6 +67,11 @@ deploy_packer = rule(
             allow_single_file = [".packer.tar"],
             doc = "`assemble_packer` label to be deployed.",
         ),
+        "overwrite": attr.bool(
+            mandatory = False,
+            default = False,
+            doc = "If overwriting the image if the same named image already exists",
```

@lriuui0x0 could be phrased simpler:

```suggestion
            doc = "Overwrite already-existing image",
```